installer/scripts/omsadmin.sh

```diff
@@ -592,6 +595,7 @@ onboard()
         echo
         echo "Generated request:"
         cat $BODY_ONBOARD
+        echo
```

Comment:
I could completely change this to a here document as it is much more readable and maintainable:
cat <<EOGREQ # or whatever

Generated request:
$BODY_ONBOARD

EOGREQ

Reply:
As you know, I'm changing how the generated request is printed in https://github.com/Microsoft/OMS-Agent-for-Linux/pull/575, and I don't think this comment applies once those changes are taken into consideration.